docs/apache-airflow/howto/connection.rst

```diff
@@ -186,6 +186,12 @@ So if your connection id is ``my_prod_db`` then the variable name should be ``AI
     Connections set using Environment Variables would not appear in the Airflow UI but you will
     be able to use them in your DAG file.
 
+.. warning::
+
```

Comment:
Can you add an explicit statement that only connections saved in the database are visible in the Web UI?